src/Components/CreateImageWizard/steps/Kernel/components/KernelName.tsx

```diff
@@ -165,7 +168,13 @@ const KernelName = () => {
         >
           <SelectList>
             {selectOptions.map((option) => (
-              <SelectOption key={option} value={option}>
+              <SelectOption
+                key={option}
+                value={option}
+                isDisabled={
+                  /not a valid kernel package name/i.test(option) ? true : false
```

Comment:
I think this is redundant - `.test` already returns a value.
```suggestion
                  /not a valid kernel package name/i.test(option)
```

Reply:
Nice catch! Updated